WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/BoldableTextView.swift

```diff
@@ -0,0 +1,113 @@
+import SwiftUI
+
+/// A text view that takes in a string whose substrings within two asterisks are bolded.
+///
+/// For example, initializing with a string "*Never* will I *stop learning*." will bold the substrings "Never" and "stop learning".
+///
+struct BoldableTextView: View {
+    fileprivate struct Element {
```

Comment:
I am curious. What would be the benefit of declaring a sort of model object in the namespace of a view?

Reply:
for private/fileprivate models, it probably doesn't matter much and it could be a separate model. I'd say it depends on the use case and scope, like for `ViewModel` it sometimes makes sense to have it inside a small struct/class like for a `UITableViewCell` while it might be better having its own struct/class when it is for a view controller.

I refactored the code here to a separate struct `BoldableTextParser` with a separate model struct `BoldableElement` that is public in https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/78d1ea47625e0becf31e7242d76036c35516120d 😄